core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -415,6 +418,34 @@ class AuthorizerIntegrationTest extends KafkaServerTestHarness {
   }
 
   @Test
+  def testPatternSubscriptionWithNoTopicAccess() {
```

Comment:
@hachikuji we are ignoring offset topic (non-matching unauthorized topic) 
testPatternSubscriptionWithTopicAndGroupRead test case. I can add separate test case, if needed.